modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -2377,7 +2378,16 @@ def _arrow_type_to_dtype(cls, arrow_type):
         object
             Any dtype compatible with pandas.
         """
-        res = arrow_type.to_pandas_dtype()
+        import pyarrow
+
+        try:
+            res = arrow_type.to_pandas_dtype()
+        except NotImplementedError as e:
```

Comment:
could you please add a comment here explaining why is this needed? this info is now present in PR message only

Reply:
added the following comment:
"Conversion to pandas is not implemented for some arrow types, perform manual conversion for them:
"